test/pdfboxing/common_test.clj

```diff
@@ -10,18 +12,28 @@
 
 (deftest obtain-document-returns-pddocument-if-provided-string-check
   (testing "obtain-document returns a PDDocument if a string path to a PDF is a supplied"
-    (is (true?  (instance?
-                   org.apache.pdfbox.pdmodel.PDDocument
-                   (obtain-document "test/pdfs/hello.pdf"))))))
+    (let [doc (obtain-document "test/pdfs/hello.pdf")]
```

Comment:
It could since we _know_ what it is, but since the _test_ is whether or not the thing is an instance of PDDocument you can't really close it if it isn't. In this case it wouldn't matter, but I was making the distinction because you had.